lib/sequent/generator/template_project/lib/account/commands.rb

```diff
@@ -0,0 +1,6 @@
+class Account
```

Comment:
Shouldn't this be a `module`?

Reply:
We reopen the class from lib/account.rb here.